library/src/main/java/com/android/volley/Request.java

```diff
@@ -29,6 +29,8 @@
 
 import java.io.OutputStream;
 import java.io.UnsupportedEncodingException;
+import java.net.MalformedURLException;
+import java.net.URL;
```

Comment:
URL is a java class that is for URLS. the android Uri class also supports things like content provider uris